tests/test_site_apportionment/test_site_apportionment.py

```diff
@@ -58,3 +58,69 @@ def test_create_notnull_mask(self):
         result = create_notnull_mask(input_df, "col")
         expected_output = self.create_exp_output()
         assert_series_equal(result, expected_output)
+
+
+class TestSortRowsOrderCols():
+
+    def create_input_df(self):
+        """Create an input dataframe for the test."""
+        input_columns = [
+            "instance",
+            "Forth",
+            "reference",
+            "Remove",
+            "period",
+        ]
+
+        data = [
+            [2, "X", 3, "test", 22],
+            [1, "X", 3, "test", 21],
+            [np.nan, "X", 3, "test", 22],
+            [2, "Y", 1, "test", 21],
+            [1, "X", 1, "test", 22],
+            [2, "X", 1, "test", 22],
+            [1, "Y", 4, "test", 21],
+            [1, "Z", 2, "test", 22],
+            [1, "X", 1, "test", 21],
+            [3, "Z", 1, "test", 22],
+
+        ]
+
+        input_df = pandasDF(data=data, columns=input_columns)
+        return input_df
```

Comment:
Is this input df going to be applicable to any more tests from site apportionment? If so, this should be converted to a pytest fixture.

Reply:
I don't think this will be used again, but I'll keep an eye out as the other tests are integrated - thanks for flagging